api/src/main/java/com/alibaba/nacos/api/config/convert/NacosConfigConverter.java

```diff
@@ -34,7 +34,7 @@
     boolean canConvert(Class<T> targetType);
     
     /**
-     * convert the Naocs's config of type S to target type T.
+     * convert the Nacos's config of type S to target type T.
```

Comment:
Would you mind change to `Convert the Nacos' config of type S to target type T.`?

Reply:
Sorry, I've changed it